<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Models</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Models</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_models">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>800</height>
           </size>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="lbl_exportPath">
            <property name="text">
             <string>Export Path</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ExportPath_Layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="exportPath"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_add_path">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_add_models">
          <property name="text">
           <string>Add Models</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_remove_models">
          <property name="text">
           <string>Remove Selected</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_rig">
        <property name="text">
         <string>Rig</string>
        </property>
        <attribute name="buttonGroup">
         <string>buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_char_anim">
        <property name="text">
         <string>Char Animations</string>
        </property>
        <attribute name="buttonGroup">
         <string>buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="rb_asset">
          <property name="text">
           <string>Asset</string>
          </property>
          <attribute name="buttonGroup">
           <string>buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chk_assetAnim">
          <property name="text">
           <string>Animations</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="chk_fixmats">
      <property name="text">
       <string>Fix Material Names</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chk_forcemats">
      <property name="text">
       <string>Force Material Export</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chk_chkOutTex">
      <property name="text">
       <string>Checkout/Add Textures to P4</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_export">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
